main.rb

```diff
@@ -1,11 +1,33 @@
 require_relative 'create'
 require_relative 'lists'
 require_relative 'colors_utils'
+require_relative 'book'
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'rental'
+require_relative 'data_related'
 
 # rubocop: disable Metrics
```

Comment:
- [OPTIONAL] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `rubocop`, it will be nice to fix the issue.